standard/expressions.md

```diff
@@ -4033,7 +4034,11 @@ where `x` is an expression of a nullable value type, if operator overload resolu
 
 ### 11.11.11 The is operator
 
-The `is` operator is used to check if the run-time type of an object is compatible with a given type. The check is performed at runtime. The result of the operation `E is T`, where `E` is an expression and `T` is a type other than `dynamic`, is a Boolean value indicating whether `E` is non-null and can successfully be converted to type `T` by a reference conversion, a boxing conversion, an unboxing conversion, a wrapping conversion, or an unwrapping conversion.
+There are two forms of the `is` operator.  One is the *is-type operator*, which has a type on the right-hand-side.  The other is the *is-pattern operator*, which has a pattern on the right-hand-side.
+
+#### 11.11.11.1 The is-type operator
+
+The *is-type operator* is used to check if the run-time type of an object is compatible with a given type. The check is performed at runtime. The result of the operation `E is T`, where `E` is an expression and `T` is a type other than `dynamic`, is a Boolean value indicating whether `E` is non-null and can successfully be converted to type `T` by a reference conversion, a boxing conversion, an unboxing conversion, a wrapping conversion, or an unwrapping conversion.
```

Comment:
This is the existing paragraph.  Such introductory paragraphs are often informal and then clarified by the text that follows.  Where I used that pattern, I introduced the text with "Informally, ".  See a couple of examples in this PR.

Reply:
I think the meaning of `compatible` in this context is clear, as it is explained in the rest of the paragraph.  If it isn't clear, we could add  the parenthetical phrase "in the sense defined below" or something of the sort after the word "compatible".  I think this can be left to be dealt with later.

Reply:
I raised it originally, so I'm happy leaving as is, unless we get feedback outside the committee.